.github/workflows/node-ci.yml

```diff
@@ -226,11 +242,15 @@ jobs:
                   git checkout $REF
 
             - name: Install & cache node_modules
+              # Only bother if there are no releases in progress
+              if: steps.check-release.outputs.release-count == '0'
               uses: ./.github/actions/shared-node-cache
               with:
                   node-version: ${{ matrix.node-version }}
 
             - name: Publish Snapshot Release to npm
+              # Only bother if there are no releases in progress
+              if: steps.check-release.outputs.release-count == '0'
```

Comment:
**note:** The job failure should mean this doesn't run, but no harm in being explicit.